Acn/Sockets/StreamingAcnSocket.cs

```diff
@@ -17,9 +17,44 @@ public class StreamingAcnSocket:AcnSocket, IProtocolFilter
         public event EventHandler<NewPacketEventArgs<StreamingAcnDmxPacket>> NewPacket;
 
         #region Setup and Initialisation
+        /// <summary>
+        /// Initializes a new instance of the <see cref="StreamingAcnSocket"/> which will unicast to a specified <see cref="IPAddress"/> on the standard ACN-SDT multicast port (5568).
+        /// </summary>
+        /// <param name="sourceId">A <see cref="Guid"/> representing this source.</param>
+        /// <param name="sourceName">A descriptive name for this endpoint.</param>
+        /// <param name="unicastDestination">The <see cref="IPAddress"/> of a remote endpoint to which packets should be directly sent.</param>
+        public StreamingAcnSocket(Guid sourceId, string sourceName, IPAddress unicastDestination)
+            : base(sourceId)
+        {
+            UnicastDestination = new IPEndPoint(unicastDestination, 5568);
+            Constructor(sourceName);
+        }
 
+        /// <summary>
+        /// Initializes a new instance of the <see cref="StreamingAcnSocket"/> which will unicast to a specified <see cref="IPEndPoint"/>. The port may be the standard ACN-SDT multicast port, or any valid port number.
+        /// </summary>
+        /// <param name="sourceId">A <see cref="Guid"/> representing this source.</param>
+        /// <param name="sourceName">A descriptive name for this endpoint.</param>
+        /// <param name="unicastDestination">The <see cref="IPEndpoint"/> to which packets should be directly sent.</param>
+        public StreamingAcnSocket(Guid sourceId, string sourceName, IPEndPoint unicastDestination)
+            : base(sourceId)
+        {
+            UnicastDestination = unicastDestination;
+            Constructor(sourceName);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="StreamingAcnSocket"/> which will multicast using standardized ports and addresses.
+        /// </summary>
+        /// <param name="sourceId">A <see cref="Guid"/> representing this source.</param>
+        /// <param name="sourceName">A descriptive name for this endpoint.</param>
         public StreamingAcnSocket(Guid sourceId, string sourceName)
             : base(sourceId)
+        {
+            Constructor(sourceName);
+        }
+
+        private void Constructor(string sourceName)
```

Comment:
Instead of a member function named Constructor, let's create a real constructor with all parameter options, and then just call that from the other overloads (this(sourceId, name, etc).